src/containers/App.tsx

```diff
@@ -115,6 +122,31 @@ function updateManifest(pathName: string): void {
     }
 }
 
+function ProgressiveWebAppPrompt(pathName: string): JSX.Element | null {
+    if (pathName === '/' || pathName.includes('admin')) return null
+    if (
+        window.matchMedia('(display-mode: standalone)').matches ||
+        document.referrer.includes('android-app://')
+    )
+        return null
+    if (getFromLocalStorage('pwaPromptShown')) return null
```

Comment:
Desse tre if-setningane kan slåast saman til éin sidan alle returnerer det same.